package.json

```
@@ -90,7 +90,7 @@
     "stmux": "stmux -nM -t addons-code-manager -e 'error,!0 errors'",
     "storybook": "start-storybook -p 9001 -c stories/setup",
     "storybook-smoke-test": "yarn storybook --smoke-test",
-    "test": "react-scripts test",
+    "test": "react-scripts test --reporters=default --reporters=`pwd`/src/jest-reporters/eslint-check.js",
```

Can we not add `\`pwd\``?

---

Sorry, I don't understand the comment.

---

Can you use a relative path instead of using the `pwd` program inlined with back ticks.

---

I tried and couldn't get it to work. I found that `pwd` trick in a SO article and it seems to be the only thing that *does* work. Do you want to try to see if you can find a different solution?

---

Did you try `<rootDir>` by chance?

```suggestion
    "test": "react-scripts test --reporters=default --reporters='<rootDir>/src/jest-reporters/eslint-check.js'",
```

---

Thanks @willdurand, that does work!